app/components/GLVolmeters.tsx

```diff
@@ -108,6 +108,7 @@ export default class GLVolmeters extends TsxComponent<VolmetersProps> {
   private sourcesOrder: string[];
   private workerId: number;
   private requestedFrameId: number;
+  // @ts-ignore
```

Comment:
Typescript thinks that `this.customizationService` is not defined here. We should use a new dependency-injector pattern to avoid this kind of error. This PR contains only changes for types to avoid the introduction of new bugs